package.json

```diff
@@ -5,18 +5,33 @@
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
     "@material-ui/core": "^3.4.0",
+    "@material-ui/icons": "^3.0.1",
+    "@mdi/font": "^3.0.39",
+    "@mdi/js": "^3.0.39",
+    "@mdi/react": "^1.1.0",
     "gatsby": "^2.0.19",
     "gatsby-image": "^2.0.15",
     "gatsby-plugin-manifest": "^2.0.5",
     "gatsby-plugin-offline": "^2.0.11",
     "gatsby-plugin-react-helmet": "^3.0.0",
-    "gatsby-plugin-sharp": "^2.0.7",
+    "gatsby-plugin-react-svg": "^2.0.0",
+    "gatsby-plugin-sharp": "^2.0.14",
     "gatsby-source-filesystem": "^2.0.4",
     "gatsby-transformer-sharp": "^2.1.4",
+    "material-auto-rotating-carousel": "^2.1.0",
+    "material-ui-icons": "^1.0.0-beta.36",
+    "mdbreact": "^4.8.4",
```

Comment:
We use [Material-UI](https://material-ui.com/) for React Material Components. Why this?

Reply:
The mdbreact one is for the carousel in the Header. Material UI does not have a carousel component. They have the slide API but the integration does not create one similar to the design. I can remove the icons as they should already be included

Reply:
Cool @puritybirir, we'll revisit it **after** first deployment but it's usually better to start with a basic implementation using Material UI library *and then* improve on it iteratively so that everybody gets the chance to see why such and such is being done. My understanding is `mdbreact` is material *library* on its own right, doesn't make sense to including it on top of Material-UI for just one component.

I also see you've included a bunch of carousel components: `material-auto-rotating-carousel`, `nuka-carousel`, `react-responsive-carousel`, etc. I take it they're no longer needed/used.